### PR TITLE
Switch fileclient to salt.utils.http.query()

### PR DESCRIPTION
**Pull request by techhat**
Ping @thatch45, @cro.

### CODE REVIEW

File: salt/utils/http.py [hunk 1]
```diff
@@ -353,12 +367,14 @@ def get_ca_bundle(opts=None):
         return opts_bundle
 
     file_roots = opts.get('file_roots', '/srv/salt')
```
Comment (s0undt3ch) [inline salt/utils/http.py]: Syspaths for the default value?

**Comment by s0undt3ch** [comment]
This seems good to me, my only question is if we want to guess in which env the users want to store the cacerts, ie, is it the best place?

Any reason why we're not storing it under `salt.syspaths. CONFIG_DIR` ?

Sorry if this has been discussed and I'm just coming Kate to the party....

**Comment by s0undt3ch** [comment]
I think I got this wrong. You're looking the file roots because it have been synced to the minions right?... Its early here ... :smile:

**Comment by techhat** [comment]
Originally I was going to write to that path to sync out to minions, but only if a default path wasn't found. But @thatch45 advised me not to just auto-pick a path for a cert.

I wouldn't have touched it in this PR, but a traceback arose which kept me from completing the task that I was actually interested in, which was to support streaming so that we could switch over fileclient.

**Pull request merged**